src/StrMap.ts

```diff
@@ -272,6 +282,14 @@ export const pop = <A>(k: string, d: StrMap<A>): Option<[A, StrMap<A>]> => {
   return a.isNone() ? none : some(tuple(a.value, remove(k, d)))
 }
 
+/**
+ * @function
+ * @since 1.4.0
+ */
+export const filter = <A>(fa: StrMap<A>, p: Predicate<A>): StrMap<A> => {
```

Comment:
since there is a `filter` method, we can avoid exporting a static function here, WDYT?

Reply:
I am fine with removing it, so do `Option` and `Either` also not reexporting it.